### DISCUSSION TITLE
Search with fd doesn't do anything

### DISCUSSION THREAD

**Discussion by sergeda** [sxyazi/yazi#1032] [1. Q&A]
When I search with fd in terminal there are bunch of files, but when I type in yazi and hit enter it just closes search input. What can I do to debug the issue?

**Comment by sxyazi** [comment]
Hi, is `fd` in your `PATH`? What does `fd --version` output? This is the `fd` parameter used by Yazi:

https://github.com/sxyazi/yazi/blob/07342a29efc974f9510840f4611b8cc0c6703879/yazi-plugin/src/external/fd.rs#L15-L21

**Comment by sergeda** [comment]
> Hi, is `fd` in your `PATH`? What does `fd --version` output? This is the `fd` parameter used by Yazi:

Yes, I have it in `PATH`:
```fd --version
fdfind 8.6.0
fd --base-directory . --regex --hidden --no-ignore build.sbt
build.sbt
```

![image](https://github.com/sxyazi/yazi/assets/914764/b70ad8b9-42c7-4cd2-ac20-4067f714613c)

**Comment by sxyazi** [comment]
Some system's `fd` is called `fdfind`, should be fixed in https://github.com/sxyazi/yazi/pull/1889
